README.md

```diff
@@ -74,3 +74,4 @@ In addition, many plugins can be found on the [npm registry](https://www.npmjs.c
 - [nova-chartjs](https://github.com/coroo/nova-chartjs) - Laravel Nova
 - [quickchart](https://github.com/typpo/quickchart) - Web API for static charts
 - [wicked-charts](https://github.com/adessoAG/wicked-charts) - Java
+- [liquify](https://github.com/sakos95/liquify) - Fast, multi-threaded visualization of stream data with Angular
```

Comment:
"Additions should be added in alphabetical order to the relevant section."

Reply:
@kurkle Thanks for pointing out and sorry for slipping on that detail.